src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -18,32 +18,34 @@
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ACTIVITY_FINISHED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ACTIVITY_STARTED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ACTIVITY_TRIGGERED;
+import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ALERT_ACKNOWLEDGED;
+import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ALERT_CEASED;
+import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ALERT_RAISED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ANNOUNCEMENT_PUBLISHED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ARTIFACT_CREATED;
+import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ARTIFACT_DEPLOYED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ARTIFACT_PUBLISHED;
+import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ARTIFACT_REUSED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.COMPOSITION_DEFINED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.CONFIDENCELEVEL_MODIFIED;
-import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.TESTCASE_CANCELED;
-import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.TESTCASE_TRIGGERED;
-import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.EXECUTION_RECIPE_COLLECTION_CREATED;
 import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.ENVIRONMENT_DEFINED;
+import static com.ericsson.eiffel.remrem.semantics.EiffelEventType.EXECUTION_RECIPE_COLLECTION_CREATED;
```

Comment:
we need all imports, so better to have * import, we can decrease no.of lines imports